worlds/sc2/rules.py

```diff
@@ -625,6 +647,35 @@ def protoss_can_attack_behind_chasm(self, state: CollectionState) -> bool:
             or (self.advanced_tactics
                 and state.has_any({item_names.ORACLE, item_names.ARBITER}, self.player))
 
+    def protoss_respond_to_colony_infestations(self, state: CollectionState) -> bool:
+        """
+        Can deal quickly with Brood Lords and Mutas in Haven's Fall and being able to progress the mission
+        :param state:
+        :return:
+        """
+        return (
+            self.protoss_common_unit(state)
+            and self.protoss_competent_anti_air(state)
+            and (
+                    state.has_any({
+                        item_names.CARRIER, item_names.SKYLORD,
+                    }, self.player)
+                    # handle mutas
+                    or state.has_any({
+                        item_names.PHOENIX, item_names.MIRAGE, item_names.SKIRMISHER,
```

Comment:
Nerfed Skirmishers don't work

Reply:
I made this change before that nerf was applied, thanks for catching it